frontend/src/pages/projects/dataConnections/const.ts

```diff
@@ -60,10 +61,10 @@ export const EMPTY_AWS_SECRET_DATA: AWSDataEntry = [
   },
   {
     key: AWS_KEYS.S3_ENDPOINT,
-    value: 'https://s3.amazonaws.com/',
+    value: '',
```

Comment:
value not needed

Reply:
value is a required field, I've removed the value's value though.

Reply:
yep, you are right. keep it as is